tests/acceptance/addon-dummy-generate-test.js

```diff
@@ -272,6 +275,13 @@ describe('Acceptance: ember generate in-addon-dummy', function() {
       assertFile('tests/dummy/app/templates/foo/bar.hbs', {
         contains: '{{outlet}}'
       });
+      assertFile('tests/dummy/app/router.js', {
+        contains: [
+          "this.route('foo', function() {" + EOL,
```

Comment:
I don't think asserting the EOL's is necessary, probably what is tripping up Appveyor.